src/meta/src/stream/stream_manager.rs

```diff
@@ -623,9 +612,32 @@ where
     /// Dropping materialized view is done by barrier manager. Check
     /// [`Command::DropMaterializedView`] for details.
     pub async fn drop_materialized_view(&self, table_id: &TableId) -> Result<()> {
+        let table_fragments = self
```

Comment:
Can we get `source_fragments` directly from a new interface inside FragmentManager? 🤔

Reply:
It doesn't seem to work, the table fragments are passed in as external parameters when creating materialized view, `fetch_source_fragments` is already the maximum level of abstraction